src/tests/comment.test.js

```diff
@@ -0,0 +1,374 @@
+    });
+    const user = await model.User.create({
+      firstname: "Maliza",
+      lastname: "Constantine",
+      telephone: "",
+      email: "malizacoco4@gmail.com",
+      password: "1234567$#8",
+      gender: "Female",
+      origin: "",
+      profession: "",
+      age: 23,
+      roleId:role.id,
+      identification_type: "",
+      identification_number: "",
+      user_image: "",
+      createdAt: new Date(),
+      updatedAt: new Date(),
+    });
+   
+    const comment = await model.Comment.create({
+      userId:user.id,
+      requesterName:`${user.firstname} ${user.lastname}`,
+      requestId: request.id,
+      comment:
+        "test seeds.....",
+      createdAt: new Date(),
+      updatedAt: new Date()
+    });
+    const tokenManager = jwt.sign({ id: user.id, roleId: user.roleId }, process.env.JWT_KEY, { expiresIn: "1h" });
+  
+    const res = await chai
+      .request(app)
+      .post(`/request/${request.id}/comment/${comment.id}/reply`)
+      .set("authorization", `Bearer ${tokenManager}`)
+    expect(res.status).to.equal(404);
+  });
+  it("should failed to reply comment for invalid request ", async () => {
+    const role = await model.userRoles.findOne({where:{name: 'MANAGER',}});
+    const request = await model.request.create({
+      idUser: 1,
+      idRoom: 1,
+      dateStart: new Date(),
+      dateEnd: new Date(),
+      requestStatus: "Pending",
+      createdAt: new Date(),
+      updatedAt: new Date(),
+    });
+    const user = await model.User.create({
+      firstname: "Maliza",
+      lastname: "Constantine",
+      telephone: "",
+      email: "malizacoco4@gmail.com",
+      password: "1234567$#8",
+      gender: "Female",
+      origin: "",
+      profession: "",
+      age: 23,
+      roleId:role.id,
+      identification_type: "",
+      identification_number: "",
+      user_image: "",
+      createdAt: new Date(),
+      updatedAt: new Date(),
+    });
+   
+    const comment = await model.Comment.create({
+      userId:user.id,
+      requesterName:`${user.firstname} ${user.lastname}`,
+      requestId: request.id,
+      comment:
+        "test seeds.....",
+      createdAt: new Date(),
+      updatedAt: new Date()
+    });
+    const tokenManager = jwt.sign({ id: user.id, roleId: user.roleId }, process.env.JWT_KEY, { expiresIn: "1h" });
+   
+    const res = await chai
+      .request(app)
+      .post(`/request/100000/comment/${comment.id}/reply`)
+      .set("authorization", `Bearer ${tokenManager}`)
+    expect(res.status).to.equal(404);
+  });
+  it("should failed to reply comment  if coment  id is invalid", async () => {
+    const role = await model.userRoles.findOne({where:{name: 'MANAGER',}});
+    const request = await model.request.create({
+      idUser: 1,
+      idRoom: 1,
+      dateStart: new Date(),
+      dateEnd: new Date(),
+      requestStatus: "Pending",
+      createdAt: new Date(),
+      updatedAt: new Date(),
+    });
+    const user = await model.User.create({
```

Comment:
You repeated creating a user many times, can you do that once and reuse her token?